### DISCUSSION TITLE
Asking users to manually create Pantry Account?

### DISCUSSION THREAD

**Discussion by SachinDas246** [imRohan/Pantry#172] [Q&A] [answered]
Like most of the APIs , If we Placing PantryID within the app ( while releasing the app to the public and not while sharing the source code), It is possible to get the PantryID from the released executable or package . Which allow that person to access and delete other user's content. One possible solution is to ask users create their own PantryID and use it within their devices. Although it is not possible to share content between users in this case, this might be helpful for certain apps . And I would like to know if asking users to create PantryID manually will violate any rules or regulations,  or negatively affect Pantry ?

**Comment by imRohan** [comment, marked as answer]
Hi @SachinDas246,

Yes you can ask users to create a Pantry account manually, which will give them their own PantryID. 

It will not violate any rules/regulations.

Enjoy!
